evil-arc-mode.el

```diff
@@ -45,10 +45,10 @@
     "gg" 'beginning-of-buffer
     "G" 'end-of-buffer
 
-    ;; goto
-    (kbd "<return>") 'archive-view
-    "o" 'archive-extract
-    "O" 'archive-extract-other-window
+    ;; follow
+    (kbd "<return>") 'archive-extract
+    (kbd "S-<return>") 'archive-extract-other-window
```

Comment:
I'm thinking, since there was nothing using 'o' and 'O' for this mode, we should keep the bindings there and also add the ones for s-return and m-return, etc.

One thing that's come to mind is that these s/m-RETURN bindings don't work in terminal. Ideally we'd have our set of bindings support terminal + GUI emacs. So I think we need another alternative to use for the terminal case.

Reply:
Yes, terminals are limited.  It should be noted that not all Emacs' default bindings work on terminals.
One that comes to mind is C-M-% (`query-replace-regexp`) on QWERTY layouts.

My suggestion: let's support terminals as much as possible, but let's not make the rationale crazy just for the sake of it.
In this case: if `o`/`O` are available, then let's bind them.

Reply:
I agree with you but, to be fair, that command C-M-% is relatively rare to do. I'm guessing the commands we're going to be replacing are somewhat more common (o -> [S-C-M] return) which makes it more of an issue.

Reply:
Maybe we should use 'go' and 'gO' also?

Reply:
That would be unnecessary if we use `gd` and `gD`, no?

Reply:
I think we should use go and gO to mirror what S-Return and M-Return would be (for terminal use).

Again, I don't necessarily believe gd == RET conceptually.

gd/gD -- go to definition related 

ret/s-ret/m-ret - related to opening things

Adding go/gO would work.

Along with that, go could be used in programming modes to 'go to other', e.g. test or header/impl files.